kotlin-styled/src/test/kotlin/styled/TestNested.kt

```diff
@@ -0,0 +1,34 @@
+package styled
+
+import kotlinext.js.asJsObject
+import kotlinx.css.Color
+import kotlinx.css.Display
+import react.RBuilder
+import react.RComponent
+import react.RProps
+import react.RState
+import kotlin.test.asserter
+
+
+class TestNested {
+    @kotlin.test.Test
+    fun testNested() {
+        class Element : RComponent<RProps, RState>() {
+            override fun RBuilder.render() {
+                styledDiv {
+                    css {
+                        display = Display.block
+
+                        firstChild {
+                            color = Color.black
+                        }
+                    }
+                }
+            }
+        }
+
+        val element = Element()
+
+        asserter.assertEquals("", "foo", element.asJsObject().toString())
```

Comment:
@Leonya What goes here?   "foo" is obviously not the expected value, it's just a placeholder.  I'm not sure this is even the right approach (using an actual ``RComponent``).

At a minimum, the test does run and fail as part of the gradle build, which was a huge pain getting to work by itself.